index.js

```diff
@@ -914,6 +915,21 @@ const generateSummary = async (testResults) => {
   return testResults
 }
 
+const generateOutputTestResultsObject = (testResults) => {
+  const outputObject = {
+    success: testResults.totalFailed === 0,
```

Comment:
Make sure your [run completion delay](https://docs.cypress.io/guides/cloud/smart-orchestration/parallelization#Run-completion-delay) is set to a high enough time so that the Cypress run does not close in between finishing the first set of specs and starting the next set.

Reply:
I don't understand this comment. It seems to be related to recording into the Cypress Cloud which is an option, but not a necessity for using this action. The run completion delay is set in the Cypress Cloud project settings not in the action, or in parameters which are passed to the action.

If a workflow does record into the cloud using the parallel parameter, then the action handles the result from each parallel container separately without attempting any consolidation. See the job [6352581480](https://github.com/cypress-io/github-action/actions/runs/6352581480) as an example where there are 3 different result summaries posted from 2 workflow jobs. The job `parallel` runs 4 specs across 2 containers. Each result summary shows 2 of the 4 specs. The job `group` summary shows all 4 spec results.

Can you explain what change to this PR you are requesting / proposing?

Reply:
Sorry, I pasted the wrong comment here completely. Not sure what happened to my original. 

@mihaisee The success value here is a calculated value that is not something that we have reflected in our Public APIs at all. We have many calculated statuses and I don't think having a success boolean calculated in this manner is reflective of all of those statuses. 

I would prefer to remove this calculated value and keep the other fields so that users can use the values they wish in the output. This should match our Public APIs like the Module API output and the after:test results output.